ruby3.4-activemodel.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: ruby3.4-activemodel
-  version: 8.0.0
-  epoch: 0
+  version: 8.0.1
+  epoch: 1
```

Comment:
not a blocker but should this be 0 because we're also bumping the version for this package.